starport/templates/ibc/oracle/dynamic/x/{{moduleName}}/types/{{queryName}}.go.plush

```diff
@@ -32,17 +32,17 @@ func NewMsg<%= queryName.UpperCamel %>Data(
 	executeGas uint64,
 ) *Msg<%= queryName.UpperCamel %>Data {
 	return &Msg<%= queryName.UpperCamel %>Data{
-		ClientID:       <%= queryName.UpperCamel %>ClientIDKey,
-		Creator:        creator,
-		OracleScriptID: int64(oracleScriptID),
-		SourceChannel:  sourceChannel,
-		Calldata:       calldata,
-		AskCount:       askCount,
-		MinCount:       minCount,
-		FeeLimit:       feeLimit,
-		RequestKey:     requestKey,
-		PrepareGas:     prepareGas,
-		ExecuteGas:     executeGas,
+		ClientID:       				<%= queryName.UpperCamel %>ClientIDKey,
+		<%= MsgSigner.UpperCamel %>:	<%= MsgSigner.LowerCamel %>,
```

Comment:
We can also check that the name doesn't collide with another field. Although, I hardly think of a developer calling the signer "SourceChannel"